rubix-prestosql/src/main/java/com/qubole/rubix/prestosql/PrestoClusterManager.java

```diff
@@ -0,0 +1,289 @@
+/**
+ * Copyright (c) 2019. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+package com.qubole.rubix.prestosql;
+
+import com.google.common.base.Throwables;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import com.google.common.collect.ImmutableList;
+import com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import com.qubole.rubix.common.utils.ClusterUtil;
+import com.qubole.rubix.spi.ClusterManager;
+import com.qubole.rubix.spi.ClusterType;
+import com.qubole.rubix.spi.thrift.ClusterNode;
+import com.qubole.rubix.spi.thrift.NodeState;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.hadoop.conf.Configuration;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
+import java.net.HttpURLConnection;
+import java.net.InetAddress;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by stagra on 14/1/16.
+ */
+public class PrestoClusterManager extends ClusterManager
+{
+  private int serverPort = 8081;
+  private String serverAddress = "localhost";
+  static LoadingCache<String, List<ClusterNode>> nodesCache;
+
+  private Log log = LogFactory.getLog(PrestoClusterManager.class);
+
+  public static String serverPortConf = "caching.fs.presto-server-port";
+  public static String serverAddressConf = "master.hostname";
+  public static String yarnServerAddressConf = "yarn.resourcemanager.address";
```

Comment:
Similarly, getResourceManagerAddress(conf) will provide yarn server address.

Reply:
Same explanation, will be removing both but only in rubix-prestosql